env0/resource_cloud_credentials_project_assignment.go

```diff
@@ -60,6 +60,11 @@ func resourceCloudCredentialsProjectAssignmentRead(ctx context.Context, d *schem
 		}
 	}
 	if !found {
+
+		if !d.IsNewResource() {
```

Comment:
Can you explain why we need to assert this one? what's the idea behind it?
Also you shouldn't be throwing here, it's either throw or remove resource from state, not both

Reply:
you right, I fix this. its need to be set to ""

Reply:
and we need to add  it because if `found `is false its mean that no credentials in that context was found, so the resource manually changed

Reply:
3868f818e1cd9e36156e9db1d2ce99f79f610410